src/ansys/additive/core/machine.py

```diff
@@ -141,6 +141,8 @@ def __eq__(self, __o: object) -> bool:
     def __validate_range(self, value, min, max, name):
         if value < min or value > max:
             raise ValueError("{} must be between {} and {}.".format(name, min, max))
+        elif math.isnan(value):
+            raise ValueError("{} must be a number.".format(name))
```

Comment:
```suggestion
        if math.isnan(value):
            raise ValueError("{} must be a number.".format(name))
        if value < min or value > max:
            raise ValueError("{} must be between {} and {}.".format(name, min, max))
```
This is really nit-picky but I think it's more common to check for `isnan()` prior to checking if a value is within limits.